src/rdkafka_partition.c

```diff
@@ -186,6 +186,22 @@ static void rd_kafka_toppar_consumer_lag_tmr_cb (rd_kafka_timers_t *rkts,
 	rd_kafka_toppar_consumer_lag_req(rktp);
 }
 
+/**
+ * @brief Enqueue an RD_KAFKA_OP_BARRIER type of operation
+ *        when the op_version is updated.
+ *
+ * @locality Toppar handler thread
```

Comment:
Also specify `@locks_required` (since rktp_op_version is written I'm guessing we need rd_kafka_toppar_lock()?)